nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/NoDnsResolverProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.nima.webclient;
+
+import io.helidon.nima.webclient.spi.DnsResolver;
+import io.helidon.nima.webclient.spi.DnsResolverProvider;
+
+/**
+ * Provider of the {@link NoDnsResolver} instance.
+ */
+public class NoDnsResolverProvider implements DnsResolverProvider {
```

Comment:
Same as above - if service loader, please add public constructor with javadocs.
Also this should contain a `@Weight` or implement `Weighted`, to have lower weight than other resolvers, so when multiple are on classpath, this one does not get used

Reply:
fixed